Documentation/gettingstarted/encryption.rst

```diff
@@ -0,0 +1,21 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gsg_encryption:
+
+************************************
+Transparent Encryption
+************************************
+
+Cilium supports the transparent encryption of traffic between Cilium-managed
+endpoints either using IPsec or Wireguard:
```

Comment:
If I'm not mistaken, IPSec also supports host-level encryption. If that's that's indeed the case, it might be worth mentioning here.